### PR TITLE
t/ckeditor5/416d: Created a "three vertical dots" icon

### PR DESCRIPTION
**Pull request by oleq**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Internal: Created a "three vertical dots" icon for the toolbar grouping feature (see ckeditor/ckeditor5#416). 

---

### Additional information

Part of https://github.com/ckeditor/ckeditor5-ui/pull/523.

### CODE REVIEW

File: theme/icons/three-vertical-dots.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg width="10" height="20" xmlns="http://www.w3.org/2000/svg"><g><circle cx="9.5" cy="4.5" r="1.5"/><circle cx="9.5" cy="10.5" r="1.5"/><circle cx="9.5" cy="16.5" r="1.5"/></g></svg>
```
Comment (dkonopka) [inline theme/icons/three-vertical-dots.svg]: We are not using `width`/`height` property for icons, let's change it to the `viewBox` (with 0 0 20 20).

 Also, I would take into consideration to transform svg content to `<path>` instead a group of `<circle>`:

Comment (oleq) [inline theme/icons/three-vertical-dots.svg]:
```suggestion
<svg viewBox="0 0 10 20" xmlns="http://www.w3.org/2000/svg"><circle cx="9.5" cy="4.5" r="1.5"/><circle cx="9.5" cy="10.5" r="1.5"/><circle cx="9.5" cy="16.5" r="1.5"/></svg>
```

Comment (oleq) [reply theme/icons/three-vertical-dots.svg]: Is path any better than circle @dkonopka?

Comment (dkonopka) [reply theme/icons/three-vertical-dots.svg]: I was thinking about the smaller weight of `.svg` file, but I'm not so sure.

**Pull request merged**